frontend/test/metabase-smoketest/admin.cy.spec.js

```diff
@@ -0,0 +1,407 @@
+import path from "path";
+import {
+  USERS,
+  restore,
+  signInAsAdmin,
+  signOut,
+  sidebar,
+} from "__support__/cypress";
+
+const admin = USERS.admin;
+const new_user = {
+  first_name: "Barb",
+  last_name: "Tabley",
+  username: "new@metabase.com",
+};
+
+describe("metabase-smoketest > admin", () => {
+  before(() => restore("blank"));
+
+  describe("Admin can setup an account", () => {
+    it("should set up Metabase", () => {
+      // This is a simplified version of the "scenarios > setup" test
+      cy.visit("/");
+      cy.findByText("Welcome to Metabase");
+      cy.url().should("not.include", "login");
+      cy.findByText("Let's get started").click();
+
+      // Language
+
+      cy.findByText("What's your preferred language");
```

Comment:
```suggestion
      cy.findByText("What's your preferred language?");
```

This doesn't seem to work without the `?`.

Reply:
Making this change actually broke the admin test when I pulled it. Where did you see that it wasn't working without the "?"

Reply:
Weird, I'll have to see if I was running off the most recent `master` branch or what...

Reply:
OK yea, I think we added the `?` at some point (see [this commit](https://github.com/metabase/metabase/commit/1f3a7680af380d7b84c3fc5ac545a2656beba610#diff-40cd5d9c20d5e224dcb6daa878617491L42), so let's add in the `?` here and when we run it against master it should be fine.

Reply:
the "?" was added in when the last test ran (8c10ee7) below